src/themes/minechat.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap");
+
+html,
+body {
+  overflow: hidden;
+  height: 100vh;
+  margin-top: 0;
+}
+
+body {
+  font-family: "Press Start 2P", cursive;
+  font-weight: 700;
+  font-size: 25px;
+}
+
+[data-twitch-chat] {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  overflow: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+[data-twitch-chat] > * {
+  margin-top: 0.6rem;
+}
+
+[data-twitch-chat] > *:last-child {
+  margin-bottom: 2rem;
+}
+
+[data-twitch-chat] [data-twitch-message] [data-twitch-message-reply] {
+  display: none;
+}
+
+/* css to add "operator crown" for the broadcaster */
+[data-twitch-sender-roles="broadcaster"] > .twitch-chat-sender::before {
```

Comment:
I think broadcasters can technically have other roles as well. To be safe, we should use the `~=` operator here to check that `broadcaster` is a substring.

Reply:
Good catch.  I am fixing that now